vendor/github.com/golang/protobuf/jsonpb/jsonpb.go

```diff
@@ -0,0 +1,1271 @@
+// Go support for Protocol Buffers - Google's data interchange format
+//
```

Comment:
kubesphere is still using dep as dependency management tool, so add dependency package to Gopkg.toml, and run `dep ensure`

Reply:
Will fix after applying go module to kubesphere project.